DigitalLearningSolutions.Web/Styles/learningMenu/section.scss

```diff
@@ -1,11 +1,11 @@
-﻿@import "~nhsuk-frontend/packages/core/all";
-
-.float-right-additional-information {
-  @include mq($until: desktop) {
-    float: none;
-  }
-
-  @include mq($from: desktop) {
-    float: right;
-  }
-}
+﻿@use "nhsuk-frontend/packages/core/all" as *;

```

Comment:
Again, this line and correcting the line endings on this file are the only changes.